EnglishFilter/sections/antiadblock.txt

```diff
@@ -550,6 +550,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75869
+||bizjournals.com/static/js/app/cxense.js
```

Comment:
Moved this to the Annoyances - `antiadblock.txt` without task comment.

Reply:
Why?

Reply:
Actually I can't reproduce it with AdGuard for Windows.

Reply:
Please check @zloyden.

Reply:
Because we can't circumvent such adb detections in regional filters.

Reply:
@zloyden Actually I can't reproduce it with AdGuard for Windows. It's blocked with the Tracking Protection filter. Please check with AdGuard for Windows.

Reply:
It's ok, if you add it without hint. It's reproduced with extensions.

Reply:
Why is it like this?

Reply:
Btw, with app it's also reproduced. Maybe dns is enabled on your end.

Reply:
Nope.

Reply:
@zloyden I only have Adguard filters except DNS and regionals. In my end, Tracking Protection Filter blocks `cxense.com`.

Reply:
> Tracking Protection Filter blocks

Here is the answer.